src/lib/Libifl/pbsD_connect.c

```diff
@@ -357,63 +248,46 @@ __pbs_connect_extend(char *server, char *extend_data)
 			setenv("SystemRoot", "C:\\WINDOWS", 1);
 		}
 #endif
-		sock = socket(AF_INET, SOCK_STREAM, 0);
+	sd = socket(AF_INET, SOCK_STREAM, 0);
+	if (sd == -1) {
+		pbs_errno = PBSE_SYSTEM;
+		return -1;
+	}
 
+	strncpy(pbs_server, server, sizeof(pbs_server)-1); /* set for error messages from commands */
```

Comment:
Use pbs_strncpy

Reply:
we should just rename pbs_strncpy -> strncpy